manifests/a/Autodesk/NavisworksManage2023/2023/Autodesk.NavisworksManage2023.yaml

```
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.9.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.9.0.schema.json
+
+PackageIdentifier: Autodesk.NavisworksManage2023

```

```suggestion
PackageIdentifier: Autodesk.NavisworksManage.2023
```

After changing the package identifier, you will also need to change the path of the manifests to `./manifests/a/Autodesk/NavisworksManage/2023/2023/*`